packages/esm-patient-attachments-app/src/camera-media-uploader/capture-photo.component.tsx

```diff
@@ -11,7 +12,16 @@ export interface CapturePhotoProps {
 
 const CapturePhoto: React.FC<CapturePhotoProps> = ({ initialState, onCapturePhoto }) => {
   const { t } = useTranslation();
+  const { patient, patientUuid } = usePatient();
   const [dataUri, setDataUri] = useState(null);
+  const isTablet = useLayoutType() === 'tablet';
+  const inEditMode = !!patientUuid;
```

Comment:
since we have the `patientUuid` in the URL when editing, am referring to it as the source truth for `inEditMode`

Reply:
any comments on this @denniskigen @vasharma05